app/components/migration/induction_record_component.html.erb

```diff
@@ -61,5 +61,9 @@
       row.with_key(text: "Training status")
       row.with_value(text: induction_record.training_status)
     end
+    list.with_row do |row|
+      row.with_key(text: "Created at")
+      row.with_value(text: induction_record.created_at)
```

Comment:
Do we want to format these GOV.UK style?

I added some formats here which can be cherry picked if useful (unless the timeline stuff is merged soon)

https://github.com/DFE-Digital/register-early-career-teachers-public/pull/297/files#diff-2cd7d9700ea89028447e936ab6d59ebd6187d3ee212db3c1c9791fdd309ff737R7

Reply:
I get the same output from no explicit format as I do from `.to_fs(:govuk)` - is that the default?
This is only so that we can see the time the objects were created for comparisons so having `2025-01-09 13:17:26 UTC` is fine